apps/meteor/ee/app/livechat-enterprise/server/settings.ts

```diff
@@ -157,6 +157,32 @@ export const createSettings = async (): Promise<void> => {
 		});
 	});
 
+	await settingsRegistry.add('Livechat_AdditionalWidgetScripts', '', {
+		type: 'string',
+		group: 'Omnichannel',
+		section: 'Livechat',
+		enterprise: true,
+		invalidValue: "",
+		multiline: true,
+		i18nLabel: 'Livechat_AdditionalWidgetScripts',
+		i18nDescription: 'Additional_script_to_embed_in_the_livechat_widget',
```

Comment:
We probably need more information on how to use the scripts, for example, it seems that scripts are separated by a comma, going by the code, maybe give an example of valid additional script

Reply:
I've added a more detailed explanation on the setting description and provided examples of usage.